app/scripts/ui/zipcode.js

```diff
@@ -0,0 +1,24 @@
+/** @jsx React.DOM */
+var React   = require('react');
+var Modal   = require('react-bootstrap/Modal');
+var Button  = require('react-bootstrap/Button');
+var Input   = require('react-bootstrap/Input');
+
+var Zipcode = React.createClass({
+    displayName: 'Zipcode',
+    render: function(){
+        return(
+            <Modal title="Enter your zipcode">
```

Comment:
Not having '-' in the id string, is it because of some naming convention we are going to follow?